src/web/core/src/main/resources/GeoServerApplication.properties

```diff
@@ -478,6 +478,7 @@ ShapefileStoreEditPanel.charset=DBF charset
 ShapefileStoreEditPanel.memoryMapped=Use memory mapped buffers (Disable on Windows)
 ShapefileStoreEditPanel.spatialIndex=Create spatial index if missing/outdated
 ShapefileStoreEditPanel.cacheMemoryMaps=Cache and reuse memory maps (Requires 'Use Memory mapped buffers' to be enabled)
+ShapefileStoreEditPanel.filterWithOdbcDbaseDriver=Use an ODBC provider to fast execution of Dbase filters (Now it only works for "Microsoft ODBC FoxPro Driver (x86)" or "Advantage StreamlineSQL ODBC driver (x86/x64)" drivers running in Windows platforms)
```

Comment:
I'd reword it to say:

Currently supports Microsoft ODBC FoxPro Driver (x86)" or "Advantage StreamlineSQL ODBC driver (x86/x64)" drivers running on Windows platform only